include/ddc/kernels/fft.hpp

```diff
@@ -581,6 +673,26 @@ void core(
 
 namespace ddc {
 
+/**
+ * @brief Initialize a discrete Fourier space.
+ *
+ * Initialize the (1D) discrete space representing the Fourier discrete dimension associated
+ * to the (1D) spatial mesh passed as argument. It is a N-periodic PeriodicSampling defined between
+ * ka=0 and kb=2*N/(b-a)*pi.
```

Comment:
```suggestion
 * Initialize the Fourier discrete dimension associated to the mesh passed as argument. It is a N-periodic PeriodicSampling defined between
 * ka=0 and kb=2*N/(b-a)*pi.
```

We are referring to `a` and `b` that are private functions.

Reply:
It seems a bit complicated, is it the same procedure as calling np.fft.fftfreq and np.fft.fftshift ?

Reply:
Short answer: yes it exactly corresponds to `fftfreq`, cf. https://github.com/CExA-project/ddc/pull/535#discussion_r1675385173